cql3/result_generator.hh

```diff
@@ -20,6 +20,7 @@ class result_generator {
     lw_shared_ptr<const query::read_command> _command;
     shared_ptr<const selection::selection> _selection;
     cql_stats* _stats;
+    bool _empty;
```

Comment:
So, our internal client couldn't handle empty pages? Irony

Reply:
It can, it currently forwards them to the client, which is actually the intended behavior.